rasa/cdu/command_processor.py

```diff
@@ -214,6 +216,18 @@ def execute_commands(
                     frame_type=StackFrameType.INTENTLESS,
                 )
             )
+        elif isinstance(command, ClarifyCommand):
+            events.append(
+                SlotSet(
+                    CLARIFICATION_OPTIONS_SLOT,
+                    command.assemble_options_string(all_flows),
```

Comment:
can we just set this to the `options` and do the "assembly" inside the pattern?  this way we can avoid hardcoding it on the command and as you suggested move it to a custom action